### PR TITLE
csrgjg/41 auth work conditional rendering back to login

### PR DESCRIPTION
**Pull request by CStewCodes**
close out #41 with commit comments outlining changes.

### CODE REVIEW

File: client/src/components/LoginUI.jsx [hunk 1]
```diff
@@ -22,90 +22,98 @@ export const LoginUI = () => {
...
+        navigate("/dashboard");
+      } else {
+        alert("Invalid email or password.");
       }
-      
-    else {
-      alert('Invalid email or password.');
+    } catch (error) {
+      console.error(error);
+      alert("An error occurred during authentication.");
```
Comment (jordan-carlson99) [inline client/src/components/LoginUI.jsx:41]: animals

Comment (joegor85) [reply client/src/components/LoginUI.jsx:41]: Strange, I thought that was from a previous commit that you made earlier, @jordan-carlson99 .

**Review by jordan-carlson99 — APPROVED** [review]
gucci

File: client/src/components/LoginUI.jsx (hunk 1 above)
Comment (jordan-carlson99) [reply client/src/components/LoginUI.jsx:41]: that doesnt sound like me -fake news

**Review by DanielPurdy509 — DISMISSED** [review]

**Comment by CStewCodes** [comment]
Looks like @joegor85 created a new branch from this PR to deconflict react hook form that @jordan-carlson99 created. We're going to have to add in the react hook form in later but for now it was causing issues with allowing us to load the login page and persist state through pages and refreshes.

**Pull request closed without merging**